### DISCUSSION TITLE
Install causing issues and unable to revert to previous settings

### DISCUSSION THREAD

**Discussion by Dearvoilas** [Rem0o/FanControl.Releases#2343] [General]
Hi, I'm struggling to revert the changes generated by this app. Even after deleting everything, fans are running full speed and are impossible to control. Even adjusting in the BIOS doesn't change anything. Any help would be appreciated. Thanks.
Edit - Anyone can advise as to how we can "cancel" the override function? TX

**Comment by Rem0o** [comment]
The changes made by the app are killed whenever you turn off/reboot your computer. Either you got it wrong in the BIOS, or you still got an app overriding the fans installed.

**Comment by Dearvoilas** [comment]
Thank you for confirming that, I appreciate it! I will try to figure out what is modulating the parameters I have. Keep up the good work!
